cypress/support/utils/arrays.js

```diff
@@ -30,6 +30,15 @@ export default {
 
     return true;
   },
+
+  checkIsSorted({ array = [], direction = 'ascending' } = {}) {
+    const result = [];
+    for (let i = 1; i < array.length; i++) {
+      result.push(array[i - 1].localeCompare(array[i]));
+    }
+
+    return direction ? result.every((n) => n <= 0) : result.every((n) => n >= 0);
```

Comment:
return the same result eiter direction = 'ascending' or 'descending' or '123'

Reply:
Fixed